content/fragments/base32-slugs.md

```diff
@@ -46,5 +46,10 @@ var lexicographicBase32Encoding = base32.NewEncoding(lexicographicBase32).
 
 So now when I list atoms from my filesystem or in S3, they come back in the same order that I wrote them.
 
+
+## Derivatives
```

Comment:
I have some custom Markdown rendering that gives headers stable anchors, but they have to be defined manually:
```suggestion
## Derivatives (#derivatives)
```